vaadin-crud-flow-parent/vaadin-crud-flow/src/test/java/com/vaadin/flow/component/crud/CrudTest.java

```diff
@@ -51,6 +49,22 @@ public void itemAvailableInAllEvents() {
                 .forEach(e -> ComponentUtil.fireEvent(systemUnderTest, e));
     }
 
+    @Test
+    public void sameItemInNewEvent() {
```

Comment:
I don't quite understand the purpose of this test.

Reply:
The thing we wanted to test is that whatever value we set for new instance (in this case a particular field), shouldn't be changed or modified inside crud component, and we expect to receive the exact same value from `getItem().field`.